contracts/CometExt.sol

```diff
@@ -72,6 +73,26 @@ contract CometExt is CometExtInterface {
 
     /** Additional ERC20 functionality and approval interface **/
 
+    /**
+     * @notice Get the ERC20 name for wrapped base token
+     * @return The name as a string
+     */
+    function name() override public view returns (string memory) {
+        uint8 i;
```

Comment:
I think `uint256` should be more gas and space efficient since it avoids an extra downscale

Reply:
Hm I just copied what we had for `symbol`, you think I should change both now?

Reply:
I'm not sure that it actually has to downscale if its all in memory (but who knows maybe it does), anyway I say we keep them as is in the name of correct intention / smaller change. Plus I'm re-deploying `kovan` already now... 😬

Reply:
Yeah not a big deal, let's keep it as is